service/controller/key/zz_generated.ami.go

```diff
@@ -210,6 +210,28 @@ var amiJSON = []byte(`{
     "us-east-2": "ami-083c654d2f0469f9e",
     "us-west-1": "ami-03efdc564552f8ed5",
     "us-west-2": "ami-0bb54692374ac10a7"
+  },
+  "2512.3.0": {
```

Comment:
How is this generated?

Reply:
Here is the docs on it: https://github.com/giantswarm/aws-operator#node-vm-images-amis
`devctl gen ami --dir service/controller/key` basically does the job :)

Reply:
Nice, didn't saw it. Thanks :)